defaults/main.yml

```diff
@@ -30,8 +30,8 @@ nginx_sendfile: "on"
 nginx_tcp_nopush: "on"
 nginx_tcp_nodelay: "on"
 
-nginx_keepalive_timeout: "65"
-nginx_keepalive_requests: "100"
+nginx_keepalive_timeout: "120"
```

Comment:
I'd rather stick to the defaults for Nginx in this role (see: https://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_timeout) — so this could be `75`, and requests could be `1000`.

Reply:
done